rkt/install.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+const (
+	rocketGroup   = "rocket"
+	groupFilePath = "/etc/group"
+)
+
+var (
+	cmdInstall = &Command{
+		Name:    "install",
+		Summary: "Sets up rocket's required directories with the right permissions",
+		Usage:   "",
+		Run:     runInstall,
+	}
+
+	// dirs relative to globalFlags.Dir
+	dirs = map[string]os.FileMode{
+		".":          os.FileMode(0755),
+		"cas":        os.FileMode(0775),
+		"tmp":        os.FileMode(0775),
+		"containers": os.FileMode(0700),
+	}
+)
+
+type Group struct {
+	Name  string
+	Pass  string
+	Gid   int
+	Users []string
+}
+
+func init() {
+	commands = append(commands, cmdInstall)
+}
+
+func parseGroupLine(line string, fields ...interface{}) {
```

Comment:
instead of doing this magic with reflection and the vararg can you just define the field indices in an enum or something and pass a `*Group` here?